alns/accept/ThresholdAcceptance.py

```diff
@@ -0,0 +1,47 @@
+from alns.accept.RecordToRecordTravel import RecordToRecordTravel
+from alns.accept.update import update
+
+
+class ThresholdAcceptance(RecordToRecordTravel):
```

Comment:
Question: why do we call this `ThresholdAcceptance`, whereas we called worse acceptance `WorseAccept`? Maybe standardise this to `ThresholdAccept`?

Reply:
I used the names as stated in Santini et al. (2018). Good idea to change it to ThresholdAccept.